deploy/kubernetes/manifests/complete-demo.yaml

```diff
@@ -0,0 +1,437 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: user
+  labels:
+    name: user
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: user
+    spec:
+      containers:
+      - name: user
+        image: weaveworksdemos/user
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: user
+  labels:
+    name: user
+spec:
+  ports:
+    # the port that this service should serve on
```

Comment:
These comments seem quite redundant, especially because we aren't doing anything worse explaining here, and it's not like we comment on other fields (if we were to attempt to explain Kubernetes).